helmfile/charts/gatekeeper-constraints/templates/restrict-latest-tag/constraint.yaml

```
@@ -0,0 +1,26 @@
+{{- if .Values.restrictTags.enable -}}
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sDisallowedTags
+metadata:
+  name: container-image-must-not-have-latest-tag
```

Since we don't know that it's the tags that's in use we should rename this
```suggestion
  name: container-image-must-not-have-disallowed-tags
```

---

Fixed